force-app/main/default/objects/Season__c/validationRules/Season_Number_Validation.validationRule-meta.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValidationRule xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Season_Number_Validation</fullName>
+    <active>true</active>
+    <description>For the same Tv Serie numbers of seasons should be uniqe</description>
```

Comment:
Put description which will be related to condition formula. For me - this rule doesn't check if Season number is unique for tv serie

Reply:
OK, is now like 'limit'